styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -55,9 +55,10 @@ $ticker-span-bg: rgba(200, 0, 0, .5);
   font-size: 29px;
   font-weight: 700;
   letter-spacing: 6px;
-  margin: 30px;
+  margin: 30px auto;
   padding: 15px;
   text-align: left;
+  width: 265px;
```

Comment:
i still dont love the idea of used a fixed pixel width and would prefer to use the make-(size)-col to vary the widths and break points. @bighappyface and @SyntaxStacks thoughts?